config/routes.rb

```diff
@@ -33,6 +33,21 @@
 
   #search users
   get '/api/v1/users/search', to: 'api/v1/users#search'
+  # test view page
+  get '/test/index', to: 'api/v1/users#stuff'
+
+  # search action
+  get '/search/action', to: 'api/v1/users#got'
+
+  # index json
+  get '/index.json', to: 'api/v1/users#pig'
+
+  #global search call
+  get :search, controller: :main
```

Comment:
What is this being used for right now?

Reply:
This the search rout that I built that calls the main controller to render the results of the last name and the first name of current users